web/src/components/MemoEditor/Editor/editor.css

```
@@ -127,3 +127,86 @@
+  font-weight: 500;
+}
+
+.cm-upload-anchor[data-status="failed"] .cm-upload-anchor-badge {
+  background: color-mix(in oklab, var(--destructive) 8%, transparent);
+  color: var(--destructive);
+}
+
+.cm-upload-anchor-spinner {
+  width: 0.75rem;
+  height: 0.75rem;
+  border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)